packages/core/admin/admin/module.d.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/no-default-export */
+declare module '*.png' {
+  const value: any;
+  export default value;
+}
```

Comment:
how does this work? it automatically types the imports for .png and svg files?

Reply:
exactly 🧠 i'm not a big fan of doing things this way but that's a refactor for another time 🤪